src/opm/input/eclipse/share/keywords/001_Eclipse300/M/MWS

```diff
@@ -5,7 +5,7 @@
   ],
   "size": {
     "keyword": "TABDIMS",
-    "item": "NUM_STATE_EQ"
+    "item": "NUM_EOS_SURFACE"
   },
   "comment": "Also note that there is no dimension for this item because there is no unit for 'molar weight' yet",
```

Comment:
You've added a unit of measurement to the reservoir condition values.  Is that not applicable to the surface condition values?

Reply:
You are right.  I just did not think about it since MWS is not introduced here. Thanks for pointing it out.